analyses/WA_cd_2010/doc_WA_cd_2010.md

```diff
@@ -0,0 +1,24 @@
+# 2010 Washington Congressional Districts
+
+## Redistricting requirements
+In Washington, districts must according to HCR 4409 (https://apps.leg.wa.gov/billsummary/?BillNumber=4409&Year=2011&Initiative=false):
+
+1. be contiguous
+2. have equal populations
+3. be geographically compact
+4. preserve county and municipality boundaries as much as possible
+
```

Comment:
you also need to mention of the special contiguity requirements: RCW 44.05.090(c): "Land areas may be deemed contiguous if they share a common land border or are connected by a ferry, highway, bridge, or tunnel. Areas separated by geographical boundaries or artificial barriers that prevent transportation within a district should not be deemed contiguous"